### PR TITLE
Add SLES 15-SP5 and drop SLES 15-SP3

### PR DESCRIPTION
**Pull request by Prabhav-Thali**
Signed-off-by: Prabhav Thali [Prabhav.Thali1@ibm.com](mailto:Prabhav.Thali1@ibm.com)
### Proposed changes
Adds SLES 15-SP5 in docs. Also, sets the generic opensuse_repo for SLES 15-SP4 and SLES 15-SP5.

### CODE REVIEW

**Comment by Prabhav-Thali** [comment]
Hi @thaJeztah,
Could you please take a look? Thanks.

File: content/engine/install/sles.md [hunk 1]
```diff
@@ -40,28 +40,23 @@ To get started with Docker Engine on SLES, make sure you
...
 - SLES 15-SP4 on s390x (IBM Z)
+- SLES 15-SP5 on s390x (IBM Z)
 
-You must enable the [`SCC SUSE`](https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.3&arch=s390x)
+You must enable the [`SCC SUSE`](https://scc.suse.com/packages?name=SUSE%20Linux%20Enterprise%20Server&version=15.5&arch=s390x)
 repositories.
 
-You must enable the [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/security). This repository is not added by default, and you need to enable
-it for the version of SLES you are running. Run the following commands to add it:
+You must enable the [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/security:/SELinux/). This repository is not added by default, and you need to enable it for the version of SLES you are running. Run the following commands to add it:
```
Comment (thaJeztah) [inline content/engine/install/sles.md:49]: This still mentions;

> and you need to enable it for the version of SLES you are running

As the instructions now seem to be version-independent; perhaps we can drop that part?

File: content/engine/install/sles.md [hunk 2]
```diff
@@ -40,28 +40,23 @@ To get started with Docker Engine on SLES, make sure you
...
 
-For SLES 15-SP3:
+For SLES 15-SP4 and SLES 15-SP5:
```
Comment (thaJeztah) [inline content/engine/install/sles.md:51]: ^^ in which case, this sentence probably can also be dropped altogether.

**Review by thaJeztah — COMMENTED** [review]
also /cc @neersighted @dvdksn

File: content/engine/install/sles.md (hunk 1 above)
Comment (dvdksn) [inline content/engine/install/sles.md:49]:
```suggestion
You must add the [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/security:/SELinux/). This repository is not added by default. Run the following commands to add it:
```

File: content/engine/install/sles.md [hunk 3]
```diff
@@ -40,28 +40,23 @@ To get started with Docker Engine on SLES, make sure you
...
 ```
 
-For SLES 15-SP4:
-
-```console
-$ opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/15.4/security:SELinux.repo"
-$ sudo zypper addrepo $opensuse_repo
-```
+If prompted to accept the GPG key, verify that the fingerprint matches
+`06B5 B9E1 5212 34E9 52FE EB7F F692 2B09 93B8 32EE`, and if so, accept it.
```
Comment (dvdksn) [inline content/engine/install/sles.md:59]: I don't like the idea of having this key verification step in the Docker docs

File: content/engine/install/sles.md [hunk 4]
```diff
@@ -40,28 +40,23 @@ To get started with Docker Engine on SLES, make sure you
...
-it for the version of SLES you are running. Run the following commands to add it:
+You must enable the [OpenSUSE `SELinux` repository](https://download.opensuse.org/repositories/security:/SELinux/). This repository is not added by default, and you need to enable it for the version of SLES you are running. Run the following commands to add it:
 
-For SLES 15-SP3:
+For SLES 15-SP4 and SLES 15-SP5:
 
 ```console
-$ opensuse_repo="https://download.opensuse.org/repositories/security:SELinux/SLE_15_SP3/security:SELinux.repo"
+$ opensuse_repo="https://download.opensuse.org/repositories/security:/SELinux/openSUSE_Factory/security:SELinux.repo"
 $ sudo zypper addrepo $opensuse_repo
```
Comment (dvdksn) [inline content/engine/install/sles.md:55]: Should there be a `zypper refresh` here?

File: content/engine/install/sles.md (hunk 1 above)
Comment (Prabhav-Thali) [reply content/engine/install/sles.md:49]: sure, we'll update as per @dvdksn's suggestion.

File: content/engine/install/sles.md (hunk 3 above)
Comment (Prabhav-Thali) [reply content/engine/install/sles.md:59]: Sure, we will remove this.

File: content/engine/install/sles.md (hunk 4 above)
Comment (Prabhav-Thali) [reply content/engine/install/sles.md:55]: We can avoid adding `zypper refresh` as while installing docker-ce packages there is a prompt to accept or reject the key.

**Comment by Prabhav-Thali** [comment]
Hi @thaJeztah @dvdksn, 
I have implemented the changes. Do let me know if anything else is required.

**Review by dvdksn — APPROVED** [review]
LGTM

**Pull request merged**